apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -130,7 +130,7 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
     public isBankUpdateInEdit: number = null;
     /** Update bank urn number */
     public updateBankUrnNumber: any;
-
+    public forceClearLinkAccount$: Observable<IForceClear> = observableOf({ status: false });
```

Comment:
Missing JS doc

Reply:
Who is responsible for this issue,,?, please fix this it today itself

Reply:
@shalinee-walkover Please ask @ravinderwalkover ji to add JS DOC